test/e2e/cluster_extension_install_test.go

```diff
@@ -319,7 +326,7 @@ func TestClusterExtensionInstallRegistryMultipleBundles(t *testing.T) {
 		assert.Equal(ct, metav1.ConditionFalse, cond.Status)
 		assert.Equal(ct, ocv1alpha1.ReasonResolutionFailed, cond.Reason)
 		assert.Contains(ct, cond.Message, "matching packages found in multiple catalogs")
-		assert.Nil(ct, clusterExtension.Status.ResolvedBundle)
+		assert.Nil(ct, clusterExtension.Status.Resolution.Bundle)
```

Comment:
This is probably where the e2e panic is
```suggestion
		assert.Nil(ct, clusterExtension.Status.Resolution)
```